ray-operator/pkg/utils/string_conversion.go

```diff
@@ -0,0 +1,17 @@
+// Util functions for type conversion between byte array and string.
+
+package utils
+
+import (
+	"unsafe"
+)
+
+// Convert a byte array to string w/o copy.
+func ConvertByteArrayToString(arr []byte) string {
```

Comment:
this seems to be a slice instead of an array

```suggestion
func ConvertByteSliceToString(arr []byte) string {
```